### ISSUE TITLE
SCSS: broken formatting with long variables

### ISSUE THREAD

**Issue by kristerkari** [prettier/prettier#2323]
Input:

```scss
$fooooooooooooooooooooooo: 2 * $baaaaaaaaaaaaaaaaaaaaaaar + ($fooooooooooooooo * 2);
```

with prettier 1.4.4
```scss
$fooooooooooooooooooooooo: 2 *
  $baaaaaaaaaaaaaaaaaaaaaaar + ($fooooooooooooooo * 2);
```

with prettier 1.5.x
```scss
$fooooooooooooooooooooooo: 2 * $baaaaaaaaaaaaaaaaaaaaaaar + ($fooooooooooooooo *
      2);
```

**Comment by kristerkari** [comment]
my config:

```
prettier --write --trailing-comma all 'src/**/*.@(ts|tsx|js|scss)'
```

**Comment by vjeux** [comment]
This is an intentional change to make most of the other pieces of CSS look better. The real solution is to implement a small arithmetic parser and printer to handle those correctly. This is tracked by https://github.com/prettier/prettier/issues/1811. I'm going to close this one, thanks for reporting it!

**Comment by kristerkari** [comment]
Alright, thanks. It is troublesome change for my project as it conflicts with [stylelint's indent rule](https://stylelint.io/user-guide/rules/indentation/) and causes linting errors:

```
message: 'Expected indentation of 0 spaces (indentation)'
at: '44,7'
source: 'stylelint'
```

**Comment by vjeux** [comment]
I would recommend turning off stylelint rules that are related to formatting, since prettier is taking over the formatting so you don't have control over them anymore.

**Comment by kristerkari** [comment]
btw. this issue is now fixed in Prettier v1.11.1
